### PR TITLE
Ender 3 Switchwire Conversion Taller Z Mod

### PR DESCRIPTION
**Pull request by buzzdalf**
* [ x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ x] Submitted STL files are oriented and scaled properly for printing.
  * [ x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Taller Z Height for Ender 3 to Switchwire Conversion
#### How was it tested? 
I tested it on my own printer.
#### Any background context you want to provide?
I am currently using this mod on my printer.

### CODE REVIEW

File: printer_mods/buzzdalf/ender_switchwire_taller_z/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+I loved the Ender 3 to Switchwire conversions that Gizzle and Triano did, but I didn't like the idea of losing z-height, so I created this mod.
+These parts are to extend the usable z-height of the printer back up around 250mm (I get 247mm with it on my own printer).  I re-used the upper Z parts Gizzle designed, and am including those here as well so you have everything you need in this repository.
+
+I also designed a bowden guide that will let you route your bowden tube between the belts and the upper cross member of the printer as seen in the picture below.  This works really well with a bowden setup, like the dual M4 extruders I use.
+
+You will need a base conversion for your Ender 3 in addition to these files.  You can find Gizzle's Ender 3 conversion in the VoronUsers github page.
```
Comment (KiloQubit) [inline printer_mods/buzzdalf/ender_switchwire_taller_z/README.md:6]: Can you add a link to that last sentence?

    You can find [Gizzle's Ender 3 conversion](../../Gizzle/ender-3_(pro)_switchwire) in the VoronUsers github page.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

- Please rename your f3d files to remove spaces from filenames.
- Add a link to Gizzle's mod as noted below.

File: printer_mods/buzzdalf/ender_switchwire_taller_z/README.md (hunk 1 above)
Comment (buzzdalf) [reply printer_mods/buzzdalf/ender_switchwire_taller_z/README.md:6]: Sure thing.  To do that, do I just go in and make the edits in my repository?  Do I have to "resubmit" anything?

Comment (KiloQubit) [reply printer_mods/buzzdalf/ender_switchwire_taller_z/README.md:6]: Just do the edits in your repository, no need to resubmit anything.  Any changes you make to your master branch will automatically show up in here.

Comment (buzzdalf) [reply printer_mods/buzzdalf/ender_switchwire_taller_z/README.md]: ok, all changes made.  Thanks for the help.

**Review by KiloQubit — APPROVED** [review]
Thanks, looks good now!

**Pull request merged**